src/ensembl/src/shared/components/instant-download/instant-download-transcript/InstantDownloadTranscriptVisualisation.tsx

```diff
@@ -128,17 +136,36 @@ const InstantDownloadTranscriptVisualisation = (props: Props) => {
     );
   });
 
-  const proteinSegments = times(exonsCount, (index) => {
-    const isOuterSegment = index === 0 || index === exonsCount - 1;
-    return (
-      <rect
-        key={index}
-        className={styles.protein}
-        x={index === 0 ? halfExonWidth : index * (exonWidth + halfExonWidth)}
-        y={exonHeight + verticalGap}
-        width={isOuterSegment ? halfExonWidth : exonWidth}
-        height={proteinHeight}
-      />
+  const proteinSegments = times(exonsCount, (si) => {
```

Comment:
`segmentIndex` and `blockIndex` is more explicit than `si` and `bi`.

Reply:
That's done